linux/setup_repo.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Where should we host this file? Is this repo good or should it be on the website?